.github/workflows/prerelease.yaml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   create_release:
     name: create release
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Any reason we don't want the latest LTS release for Ubuntu? Or should we support both, as 20.04 is still well within the support period?

Reply:
The only reason is better compatibility with older versions. This is the same reason I didn't configure the latest Windows and macOS versions.

Reply:
Sounds reasonable to me. I just doubled checked with our other repositories and runners are all over the place, which seems like we should address that internally. Just to name a few:

- Jazzer: [ubuntu-20.04, macos-12, windows-2019]
- Jazzer.js (with thisPR): [ubuntu-20.04, macos-11, windows-2019]
- cifuzz: [ubuntu-22.04, macos-12, windows-2022]
- gofuzz: [ubuntu-latest, macos-latest, windows-latest]

Regardless of the different projects == different requirements' topic, as far as I can tell, we have no indicator (facing publicly), which platforms we want to support. WDYT?

Reply:
Sounds good to me

Reply:
Could you approve this PR?